Dockerfile

```diff
@@ -19,6 +19,9 @@ WORKDIR /app
 RUN adduser -u 1001 --disabled-password --gecos "" appuser && \
     chown appuser:appuser /app
 
+RUN mkdir -p /app/data && \
+    chown -R appuser:appuser /app/data
```

Comment:
merge chown from prev instruction and from current one

Reply:
done

Reply:
If we need to give appuser permissions recursively to all directories/files inside /app (not only in /data), then I can leave only one

Reply:
done

Reply:
still see 2 chown commands in the same RUN block